### ISSUE TITLE
Color Contrast Fails for Social Media Icons (Grey Icons on White Background Below 3:1 Ratio): A11y_ Visual Studio Code Services_Open Source AI code editor screen _Contrast Minimum.

### ISSUE THREAD

**Issue by AnKushSingh05** [microsoft/vscode-docs#9357]
### GitHub Tags: 
#A11yTCS; #A11yMAS; #A11ySev2; #AI Limited; #DesktopWeb; #ChromiumEdge; #Win11; #Visual Studio Code Services; #E2E_Visual Studio Code Services_Web_Jan26; #WCAG1.4.3; #Focus Order; #ES Chat;   

"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after bug verification." 

Try [ES Chat](https://nam06.safelinks.protection.outlook.com/?url=http%3A%2F%2Faka.ms%2Feschat-ask%3Fq%3DDoes%2BMAS%2B1.4.3%2Bapply%2Bto%2Belements%2Bin%2Bother%2Bstates%2B(such%2Bas%2Bfocus%2Bor%2Bhover)%3F&data=05%7C02%7Cv-yaja%40microsoft.com%7Cab1c8bf7fc40479470f408de5d149d45%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C639050540322712738%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&sdata=h0pesliWcxZ8OIISTmTyucruKF1rvvoJGsCM7BdUOa0%3D&reserved=0) to learn more about the MAS rule and how to fix the issue. If you need more help, use our [Teams channel](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2fteams.microsoft.com%2fl%2fchannel%2f19%253ADeCtYeREyQHw5jYsz9PPc3hpVw48m-tOlTkQ1jEn0eM1%2540thread.tacv2%2fCAI%2520Accessibility%2520Support%2520Channel?groupId%3Df5eb428c-9e6e-4c16-8143-33fe3961a9ae%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&data=05%7c02%7cv-pawanksing%40microsoft.com%7c26cff27270c448f48a0e08de3f569146%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c639017838221198042%7cUnknown%7cTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7c0%7c%7c%7c&sdata=DCNVsZe8oDZWqm97AI0ee4OoWWecR6zw6qlhTuzP7MI%3D&reserved=0) or [office hours](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2foutlook.office.com%2fbook%2fmicrosoft2779007%2540microsoft.onmicrosoft.com%2fs%2fbLeGBltMW0qI0WRLsTqx2A2?ismsaljsauthenabled&data=05%7c02%7cv-pawanksing%40microsoft.com%7c26cff27270c448f48a0e08de3f569146%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c639017838221214666%7cUnknown%7cTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7c0%7c%7c%7c&sdata=CnCCdDgEmll2oUQUdMjXusAgV8PneqZVYvzgxAfk7Ik%3D&reserved=0). 
### Environment Details: 
Application Name: Visual Studio Code Services 
URL: [https://code.visualstudio.com](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fcode.visualstudio.com%2F&data=05%7C02%7Cv-kduvvuri%40microsoft.com%7Cf492af0a8c204ceb833c08de5d7335b8%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C639050947034495318%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&sdata=pGQTzxzKOp7hs0ceHAv0XoOcNW3%2Byeh5%2F%2FMFLbSZonA%3D&reserved=0)  
Chromium Edge: Version 144.0.3719.104 (Official build) dev (64-bit) 
Edition: Windows 11 Enterprise Insider Preview 
Version: 24H2 
OS build: 26558.1000 
### Repro steps: 

1. Hit the URL: [https://code.visualstudio.com](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fcode.visualstudio.com%2F&data=05%7C02%7Cv-kduvvuri%40microsoft.com%7Cf492af0a8c204ceb833c08de5d7335b8%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C639050947034495318%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&sdata=pGQTzxzKOp7hs0ceHAv0XoOcNW3%2Byeh5%2F%2FMFLbSZonA%3D&reserved=0) 
2. Tan navigate to Social media icons at bottom of the screen. 
3. Turn on Accessibility Insights color contrast and Observe that Color Contrast Fails for Social Media Icons (Grey Icons on White Background Below 3:1 Ratio). 

### Actual Result: 
The social media icons displayed in grey on a white background have a contrast ratio that is less than 3:1, making them hard to distinguish for users with visual impairments or in environments with poor lighting. 
### Expected Result: 
The social media icons should have a contrast ratio of at least 3:1 against the white background to ensure they are discernible for all users, particularly those with low vision.
### User Impact: 
Low vision users may experience challenges if the color contrast for social media icons does not meet accessibility standards, such as using grey icons on a white background with a contrast ratio below 3:1. This insufficient contrast can make it difficult for these users to distinguish the icons, potentially affecting their ability to navigate or interact with the content. Ensuring adequate color contrast is important to support accessibility and usability for everyone. 
### Attachment:

<img width="960" height="540" alt="Image" src="https://github.com/user-attachments/assets/de28648d-eab3-41dd-a6e8-1d62dd603904" />

**Comment by ntrogh** [comment]
@AnKushSingh05 Issue should be resolved - can you retest?

**Comment by AnKushSingh05** [comment]
Issue is fixed

<img width="1280" height="720" alt="Image" src="https://github.com/user-attachments/assets/edc46a02-a944-4906-97fe-6d467d5ed9f8" />
